pydantic/deprecated/config.py

```diff
@@ -18,10 +18,11 @@
 
 class _ConfigMetaclass(type):
     def __getattr__(self, item: str) -> Any:
-        warnings.warn(_config.DEPRECATION_MESSAGE, DeprecationWarning)
 
```

Comment:
```suggestion
```

Looks like this is all that's needed to get linting to pass so that this can be merged.

Reply:
ah whoops!  sorry this fell off my radar